processor/otel/consumer.go

```diff
@@ -687,6 +692,22 @@ func statusCodeResult(statusCode int) string {
 	return fmt.Sprintf("HTTP %d", statusCode)
 }
 
+// serverStatusCodeOutcome returns the transaction outcome value to use for the given status code.
+func serverStatusCodeOutcome(statusCode int) string {
+	if statusCode >= 500 {
+		return "failure"
```

Comment:
consider making these `const`, maybe in shared package if used elsewhere

Reply:
Added constants to the top of the file. Maybe we can move them to model later.